components/Post/PageProgress.tsx

```diff
@@ -34,7 +38,7 @@ const PageProgress = () => {
         className="progress-bar"
         style={{
           width: 0,
-          transition: "100ms",
+          transition: "width 200ms",
```

Comment:
need this one, because on mobile version the progress bar is lagging little bit and want to be more smoother